index.js

```diff
@@ -57,7 +60,7 @@ module.exports = function (options, logger) {
 					cb();
 				}
 			})
-			.create();
+			.create({}, options);
```

Comment:
`stampit` has a weird API... Hopefully I can land a way to really lint a string in `stylint` soon